docs/tutorials/tabular_prediction/tabular-quickstart.md

```diff
@@ -76,20 +79,67 @@ AutoGluon automates this process.
+print(predictor.feature_metadata)
+```
+
+AutoGluon correctly recognized our prediction problem to be a **binary classification** task and decided that variables such as `age` should be represented as integers, whereas variables such as `workclass` should be represented as categorical objects. The `feature_metadata` attribute allows you to see the inferred data type of each predictive variable after preprocessing (this is it's *raw* dtype; some features may also be associated with additional *special* dtypes if produced via feature-engineering, e.g. numerical representations of a datetime/text column).
+
+We can evaluate the performance of each individual trained model on our (labeled) test data:
+```{.python .input}
+predictor.leaderboard(test_data, silent=True)
+```
+
+When we call `predict()`, AutoGluon automatically predicts with the model that displayed the best performance on validation data (i.e. the weighted-ensemble). We can instead specify which model to use for predictions like this:
+```
+predictor.predict(test_data, model='NeuralNetClassifier')
+```
+
+Above the scores of predictive performance were based on a default evaluation metric (accuracy for binary classification). Performance in certain applications may be measured by different metrics than the ones AutoGluon optimizes for by default. If you know the metric that counts in your application, you should specify it as demonstrated in the next section.
+
+## Maximizing predictive performance
+
+To get the best predictive accuracy with AutoGluon, you should generally use it like this:
+
+```{.python .input}
+time_limits = 60 # for quick demonstration only, you should set this to longest time you are willing to wait (in seconds)
+metric = "roc_auc" # specify your metric here
+predictor = task.fit(train_data=train_data, label=label_column, time_limits=time_limits,
+                     eval_metric=metric, presets='best_quality')
 ```
 
-AutoGluon correctly recognized our prediction problem to be a binary classification task and decided that variables such as `age` should be represented as integers, whereas variables such as `workclass` should be represented as categorical objects.
+This command implements the following strategy to maximize accuracy:
+
+- Specify the argument `presets='best_quality'`, which allows AutoGluon to automatically construct powerful model ensembles based on [stacking/bagging](https://arxiv.org/abs/2003.06505), and will greatly improve the resulting predictions if granted sufficient training time. The default value of `presets`is `'medium_quality_faster_train'`, which produces *less* accurate models but facilitates faster prototyping. With `presets`, you can flexibly prioritize predictive accuracy vs. training/inference speed. For example, if you care less about predictive performance and want to quickly deploy a basic model, consider using: `presets=[‘good_quality_faster_inference_only_refit’, ‘optimize_for_deployment’]`.
```

Comment:
The default value of `presets`is ->  The default value of `presets` is

Missing space between `presets` and `is`